packager/packager.cc

```diff
@@ -1259,6 +1283,19 @@ struct PackageSpecBodyWalk {
     }
 
 private:
+    string buildValidLayersStr(const core::GlobalState &gs) {
+        auto &validLayers = gs.packageDB().layers();
+        ENFORCE(validLayers.size() > 0);
+        if (validLayers.size() == 1) {
+            return string(validLayers.front().shortName(gs));
+        }
+        string result = "";
+        for (int i = 0; i < validLayers.size() - 1; i++) {
+            result += fmt::format("{}, ", validLayers[i].shortName(gs));
+        }
+        result += fmt::format("or {}", validLayers.back().shortName(gs));
```

Comment:
We might want to use `ErrorColors::format` here to get colorized formatting for the layer names.